libbpf-rs/src/program.rs

```diff
@@ -36,6 +36,12 @@ impl OpenProgram {
             libbpf_sys::bpf_program__set_ifindex(self.ptr, idx);
         }
     }
+
+    /// Get program section name
+    pub fn section_name(&self) -> Result<String> {
```

Comment:
hm... just looked at Program and it seems like it has similar accessor, but it's called just `section` and it returns `&str` (because it creates internal copy). We should probably keep them consistent, no?

Reply:
Do you mean the name should be the same? if so then sure, I'll make the change, as for the copy, if you will refer to how it work with Program, it's actually a member in the struct that got there when it was constructed
```
            // Get the program section
            // bpf_program__section_name never returns NULL, so no need to check the pointer.
            let section = unsafe { libbpf_sys::bpf_program__section_name(next_ptr) };
            let section = util::c_ptr_to_string(section)?;

            // Get the program fd
            let fd = unsafe { libbpf_sys::bpf_program__fd(next_ptr) };
            if fd < 0 {
                return Err(Error::System(-fd));
            }

            // Add the program to the hashmap
            obj.progs
                .insert(name.clone(), Program::new(next_ptr, name, section));
            prog = next_ptr;
```

Reply:
I meant both the name and return type. OpenProgram could do the same thing as Program, right? Are there any downsides to that?

Reply:
Done